src/main/java/org/jenkinsci/plugins/workflow/support/pickles/ComputerPickle.java

```diff
@@ -34,8 +34,9 @@
 /**
  * Reference to {@link Computer}
  *
- * @author Kohsuke Kawaguchi
+ * @deprecated Normally now done via {@link ExecutorStepDynamicContext}.
  */
+@Deprecated
```

Comment:
Can the factories for these pickles be deleted or replaced with an implementation that just throws an exception if it encounters an object of the specified type? Or do you want to keep them around in case some other step or wild Groovy code is relying on them?

Reply:
I plan to retain them for compatibility. Weird Pipeline script is one possibility. They could also be used in other Pipeline steps; I know `PushdStep` binds `FilePathDynamicContext`, so that is not affected, but there may be others. Seems harmless enough to leave them here.